platform/ui-next/src/components/ThumbnailList/ThumbnailList.tsx

```diff
@@ -10,7 +10,8 @@ const ThumbnailList = ({
   onClickUntrack,
   activeDisplaySetInstanceUIDs = [],
   viewPreset,
-}) => {
+  commandsManager,
```

Comment:
No

Reply:
UI should be dummy, no services and no commands, if you need to run a command pass it as props, onWhatEverClick = {}